src/components/PupilComponents/QuizResultMCQ/QuizResultMCQ.tsx

```diff
@@ -16,14 +16,24 @@ import { MathJaxWrap } from "@/browser-lib/mathjax/MathJaxWrap";
 export type QuizResultMCQProps = {
   answers: MCAnswer[];
   feedback: QuestionFeedbackType[];
+  pupilAnswer: number[];
 };
 
-export const QuizResultMCQ = ({ answers, feedback }: QuizResultMCQProps) => {
+export const QuizResultMCQ = ({
+  answers,
+  feedback,
+  pupilAnswer,
+}: QuizResultMCQProps) => {
+  const mappedFeedback = feedback.map((mark, index) => {
+    if (pupilAnswer.includes(index)) {
+      return mark;
+    } else return null;
```

Comment:
`}else{
return null
}`

or use a ternary and get rid of curly braces altoghether